app/components/Layout/SubHeader.tsx

```diff
@@ -27,7 +27,14 @@ export default function SubHeader(props: Props) {
     <BaseHeader header="sub">
       <ul>
         <li>
-          <Link href="/cif">
+          <Link
+            href={
+              // The external user subheader doesn't contain an operators link, so we can use its presence/absence to check if we're routing internal vs. external users
+              props.links.some((link) => link.name === "Operators")
+                ? "/cif"
+                : "/cif-external"
+            }
```

Comment:
Just a suggestion:
```
            href={
              router.pathname.includes("/cif-external")
                ? "/cif-external"
                : "/cif"
            }

Reply:
I think this is a good suggestion, but changing it will require some test updates and I'd like to get this one in sooner than later to avoid merge conflicts for @gurjmatharu. (Also first slice routing is probably temporary so this code may not last long)

Reply:
Sure